providers/fab/src/airflow/providers/fab/auth_manager/security_manager/override.py

```diff
@@ -538,6 +543,10 @@ def create_jwt_manager(self):
         jwt_manager.init_app(self.appbuilder.app)
         jwt_manager.user_lookup_loader(self.load_user_jwt)
 
+    def refresh_jwt_token(self) -> str:
```

Comment:
I dont think we should use FAB to manage the JWT token for us. We should do it ourselves, the same way we do it with simple auth manager. That would be way simpler. We just need to get the user from user credentials passed from the API. Once we get that user we can call `auth.manager.get_jwt_token()`. Very similarly to simple auth manager

Reply:
Make sense. Good pointer! Let me make the changes. Thanks! :)

Reply:
Updated, I also reused the BaseAuthManager signer in SimpleAuthManager, seems they are the same thing defined in different places.